tests/console_test.go

```diff
@@ -229,7 +230,7 @@ var _ = Describe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@redha
 				Expect(virtClient.RestClient().Post().Resource("virtualmachineinstances").Namespace(tests.NamespaceTestDefault).Body(vmi).Do().Error()).To(Succeed())
 
 				By("Expecting the VirtualMachineInstance console")
-				_, _, err := tests.NewConsoleExpecter(virtClient, vmi, 30*time.Second)
+				_, _, err := console.NewConsoleExpecter(virtClient, vmi, 30*time.Second)
```

Comment:
We can rename it to `console.NewExpecter` now that we have `console` as the package name.